docs/testbed/README.testbed.VsSetup.md

```diff
@@ -101,7 +101,19 @@ cd sonic-mgmt
 ./setup-container.sh -n <container name> -d /data
 ```
 
-2. From now on, **all steps are running inside the sonic-mgmt docker**, unless otherwise specified.
+2. (Required for IPv6 test cases): Follow the steps [IPv6 for docker default bridge](https://docs.docker.com/config/daemon/ipv6/#use-ipv6-for-the-default-bridge-network) to enable IPv6 for container. Here is an example "/etc/docker/daemon.json" which uses ULA address. Then restart docker daemon `sudo systemctl restart docker` to take effect.
```

Comment:
Please add one more statement here like saying, just run the 
```
echo .....
EOF
```
to set up the docker daemon file if no special requirement or no local docker configuration, to simplify this step

Reply:
Updated the description and added the shell procedure. Please review it again, and let me know any comments.